.github/workflows/publish.yml

```
@@ -43,14 +43,14 @@ jobs:
     name: Build Wheel
     runs-on: ${{ matrix.os }}
     needs: release
-    
+
     strategy:
       fail-fast: false
       matrix:
           os: ['ubuntu-20.04']
           python-version: ['3.8', '3.9', '3.10', '3.11']
-          pytorch-version: ['2.0.1']
-          cuda-version: ['11.8'] # Github runner can't build anything older than 11.8
+          pytorch-version: ['2.1.0']
+          cuda-version: ['12.2'] # Github runner can't build anything older than 11.8
```

```suggestion
          cuda-version: ['12.2']
```
Could you check whether this works with CUDA 12.2?

---

@zhuohan123 Did you check this by any chance?